modules/lvms-provisioning-storage-using-logical-volume-manager-operator.adoc

```diff
@@ -4,67 +4,75 @@
+After you have created the LVM volume groups using the `LVMCluster` custom resource (CR), you can provision the storage by creating persistent volume claims (PVCs).
 
-[NOTE]
-====
-{lvms} provisions PVCs in units of 1 GiB. The requested storage is rounded up to the nearest GiB.
-====
+To create a PVC, you must create a `PersistentVolumeClaim` object.
+
+.Prerequisites
+
+* You have created an `LVMCluster` CR.
 
 .Procedure
 
-. Identify the `StorageClass` that is created when {lvms} is deployed.
-+
-The `StorageClass` name is in the format, `lvms-<device-class-name>`.
-The `device-class-name` is the name of the device class that you provided in the `LVMCluster` of the `Policy` YAML.
-For example, if the `deviceClass` is called `vg1`, then the `storageClass` name is `lvms-vg1`.
-+
-The `volumeBindingMode` of the storage class is set to `WaitForFirstConsumer`.
+. Log in to the OpenShift CLI (`oc`).
 
-. To create a PVC where the application requires storage, save the following YAML to a file with a name such as `pvc.yaml`.
+. Create a `PersistentVolumeClaim` object:
 +
-.Example YAML to create a PVC
+.Example `PersistentVolumeClaim` object
 [source,yaml]
 ----
-# block pvc
-apiVersion: v1
-kind: PersistentVolumeClaim
-metadata:
-  name: lvm-block-1
-  namespace: default
-spec:
-  accessModes:
-    - ReadWriteOnce
-  volumeMode: Block
-  resources:
-    requests:
-      storage: 10Gi
-  storageClassName: lvms-vg1
----
-# file pvc
 apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
-  name: lvm-file-1
+  name: lvm-block-1 <1>
   namespace: default
 spec:
   accessModes:
     - ReadWriteOnce
-  volumeMode: Filesystem
+  volumeMode: Block <2>
   resources:
     requests:
-      storage: 10Gi
-  storageClassName: lvms-vg1
+      storage: 10Gi <3>
+  storageClassName: lvms-vg1 <4>
 ----
+<1> Specify a name for the PVC.
+<2> To create a block PVC, set this field to `Block`. To create a file PVC, set this field to `Filesystem`.
+<3> Specify the storage size. {lvms-first} provisions PVCs in units of 1 GiB (gibibytes). The requested storage is rounded up to the nearest GiB. The total storage size you can provision is limited by the size of the LVM thin pool and the overprovisioning factor.
+<4> The value of the `storageClassName` field must be in the format `lvms-<device_class_name>` where `<device_class_name>` is the value of the `deviceClasses.name` field in the `LVMCluster` CR.
+For example, if the `deviceClasses.name` field is set to `vg1`, you must set the `storageClassName` field to `lvms-vg1`.
++
+[NOTE]
+====
+The `volumeBindingMode` field of the storage class is set to `WaitForFirstConsumer`.
+====
 
 . Create the PVC by running the following command:
 +
 [source,terminal]
 ----
-# oc create -f pvc.yaml -ns <application_namespace>
+# oc create -f <file_name> -n <application_namespace>
```

Comment:
I see that the `#` is existing, but this seems off compared to every instance of running `oc` I have ever encountered. Are you 100% sure it's not intended to be `$`? Just want to verify since it's an outlier :)

Reply:
This is a great catch! It is actually `$`. 
Updated accordingly. 
Thanks!